### ISSUE TITLE
Unable to access Komga after upgrade to v1.3.0 running in Docker container on Synology

### ISSUE THREAD

**Issue by SoccerNerd** [gotson/komga#1185]
### Steps to reproduce

Have been running with no issues for several months. Yesterday upgraded Komga container to version 1.3.0 and now when trying to access Komga interface I get a "connection reset" error message in my browser. 

Container starts without any issues and looking at the container log and I am not seeing any errors (see attached) 

### Expected behavior

I should see the Komga login page

### Actual behavior

"connection reset" error message in browser

### Logs

see attached
[komga.csv](https://github.com/gotson/komga/files/12262609/komga.csv)

### Komga version

1.3.0

### Operating system

Synology DSM 7.2-64570 Update 1

### Other details

Docker Container Manager 20.10.23-1413

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have checked the [FAQ](https://komga.org/faq/).
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by SoccerNerd** [comment]
Adding that I can also not access my Komga library from Panels application on iPad connected to Panes as OPDS source. This was also previously working

**Comment by DieselTech** [comment]
Just note that your not on 1.3.0. The logs show you on 1.2.1. The container for 1.3.0 wasn't released yesterday due to build problems. 

How are you accessing the web interface? What is the URL you use?

**Comment by SoccerNerd** [comment]
DOH! I missed that in the logs. 

I pulled the latest image from the registry this morning and it now shows 1.3.0, but still getting the same error.  

My container is listening on port 8341 and I'm accessing using HOSTNAME:8341. I've also tried the IP address instead and both HTTP and HTTPS (I have a cert configured on my Synology system). Same result. 

Latest log attached. 
[komga_2.csv](https://github.com/gotson/komga/files/12263296/komga_2.csv)
